### DISCUSSION TITLE
`chatting` with individual user in private?

### DISCUSSION THREAD

**Discussion by oyetanishq** [community/community#36026] [Discussions]
## **Overview**

1. I don't trust `fb`, `ig`, `wb`
2. Maybe we can give individual chat an option for `custom encryption`
3. Can use `markdown` in chat.
4. Hell lot of things we can do here, and I'll get my whole family to github if this happens 😂

**Comment by Animeshsingh17** [comment]
I'll Convey the same message to my family through an encrypted message 👍

**Comment by VasylHulpak** [comment]
May i ask you a question: why do you need individual chat ? we all use other social media for chatting. I guess the main purpose of hitub is storage and help other people on their issues, gave solution (i mean implemented own packages etc)
